load_data.py

```diff
@@ -61,7 +61,7 @@ def create_image_array(image_list, image_path, nr_of_channels):
 
 # If using 16 bit depth images, use the formula 'array = array / 32767.5 - 1' instead
 def normalize_array(array):
-    array = array / 127.5 - 1
+    array = array /255

```

Comment:
The current division solves several issues with the histogram of output images, as well as training convergence.